frontend/src/routes.tsx

```diff
@@ -76,17 +76,23 @@ export default function AppRoutes () {
         auth.signinRedirect,
     ]);
 
-    useMemo(() => {
-        dispatch(getConfiguration({configScope: 'global'}));
+    useMemo(async () => {
+        let configFetched = false;
+        if (!configFetched) {
+            await dispatch(getConfiguration({configScope: 'global'}));
+        }
+        
         if (configLoadError) {
             notificationService.generateNotification(
                 'Error loading app configuration. Restrictive default policy has been applied in its place. Consult with system admin to resolve issue.',
                 'error'
             );
         }
+        return () => {
```

Comment:
I don't think you need to return anything since you aren't setting this to a value.